### DISCUSSION TITLE
when implementing botD with dashboard

### DISCUSSION THREAD

**Discussion by noa55345** [fingerprintjs/BotD#59] [General]
when implementing botD with dashboard we don't need to implement botD on every page (using next js project), only we do it with the login page and sign up(cause bots cant access other pages without a session) should I do it like that or implement it on all pages (using full protection).

**Comment by Romchela** [comment]
Hi! That's ok to use BotD only on pages where it's required.

**Reply by noa55345** [reply]
sorry for the extended discussion, if I enabled botD on login and sign In pages without the dashboard itself when the malicious user access the app(legit) after login in or sing up, can he use malware bot to hit other endpoint need session (need authentication), I mean in this case I should use botD on every page neither need a session to access them nor not, right?
Another question is how do I know what type of malicious bots when receiving the result(malicious  bots can detect if automationTool =1 or browserSpoofing = 1, etc) maybe I let some not malicious bots access, thanks

**Reply by Romchela** [reply]
1) You need to do BotD call when client does any actions that you want to forbid for automation tools.
2) automationTool>0.5 when client uses selenium, puppeteer, etc. browserSpoofing >0.5 when user tries to spoof some browser parameters (e.g. user agent). It depends on your use case what condition should you implement based on our response.
